dashboard/test/ui/features/acquisition_products/sign_up.feature

```diff
@@ -0,0 +1,22 @@
+@eyes
+Feature: Teacher can create a new account
+
+Scenario: Teacher can create an account with the new school association flow
+
+  Given I am on "http://studio.code.org/users/sign_up"
+  And I press keys "my@validemail.com" for element "#user_email"
+  And I press keys "password" for element "#user_password"
+  And I press keys "password" for element "#user_password_confirmation"
+  And I press "#signup_form_submit" using jQuery
+  # Add the experiment flag
+  And I am on "http://studio.code.org/users/sign_up?enableExperiments=school_association_v2"
+  And I press "select-user-type-teacher"
+  And I press keys "myDisplayName" for element "#user_name"
+  And I select the "United States" option in dropdown "uitest-country-dropdown"
+  And I press keys "98112" for element "#uitest-school-zip"
+  And I select the "Holy Names Academy" option in dropdown "uitest-school-dropdown"
```

Comment:
Is this school seeded in test?

Reply:
Not me, falling under the Saucelabs spell and forgetting that it's still my local data even though it's the same software we use for drone🤦🏻‍♀️

Reply:
I think your update is fine but if you wanted to select a real school, we do have a handful in test. This is the fixture file: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/fixtures/schools.tsv

Reply:
Got it to work with a real school! Resolving